docs/data-ai/reference/api/data-client.md

```diff
@@ -4,5 +4,7 @@ linkTitle: "Data client API"
 weight: 30
 type: "docs"
 layout: "empty"
+aliases:
+  - /data-ai/reference/data-client/
```

Comment:
Because this is not a real page but only something used for forwarding you can skip these aliases. It's very unlikely we'll get 404s from those.